boards/opentitan/README.md

```diff
@@ -109,23 +109,80 @@ export OPENTITAN_TREE=/home/opentitan/
 
 then you can run `make flash` or `make test-hardware` to use the board.
 
-### Compiling the Kernel for FPGA or Verilator
+Verilator
+---------
 
 Opentitan is supported on both an FPGA and in Verilator. Slightly different
 versions of the EarlGrey chip implementation are required for the different
-platforms. By default the kernel is compiled for the FPGA. To compile for
-Verilator, run:
+platforms. By default the kernel is compiled for the FPGA.
+
+## Setting up Verilator
+
+For a full guide see the official OpenTitan documentation: https://docs.opentitan.org/doc/ug/getting_started_verilator/
+
+A quick summary on how to do this is included below though
+
+### Build FuseSoc
+
+```shell
+git clone https://github.com/lowRISC/opentitan.git
+cd opentitan
+git checkout <OpenTitan_SHA>
```

Comment:
Can we add a comment telling users how to find a SHA they can reasonably expect to work here? The top-level Makefile downloads a particular release but its not immediately clear to me how to go from that release to a git SHA. Should we putting the tested/supported OT SHA somewhere?

Reply:
It is specified in the top of this README